resources/assets/sass/regions/_user.scss

```diff
@@ -0,0 +1,217 @@
+.r-user {
+
+    background: $color-fill-neutral-light;
+}
+
+.r-user__header {
+
+    overflow: hidden;
+}
+
+.r-user__info {
+
+    @include color-modifiers-and(background);
+
+    position: relative;
+    padding-bottom: $spacer * 4;
+}
+
+    .r-user__map {
+
+        position: absolute;
+        bottom: $spacer * 4;
+        z-index: 1;
+       display: none;
+
+        @include respond-to(tablet) {
+
+            right: -$spacer * 8;
+            display: block;
+            width: 100%;
+        }
+
+        @include respond-to(desktop) {
+
+            right: -$spacer * 8;
+            width: 90%;
+        }
+
+        @include respond-to(desktop-large) {
+
+            right: $spacer * 4;
+            width: 80%;
+        }
+    }
+
+    .r-user__info-wrap {
+
+        @include wrap('narrow');
+
+        position: relative;
+        z-index: 2;
+    }
+
+        .r-user__info-image {
+
+            @include color-modifiers-and(border-color);
+
+            position: relative;
+            top: -$spacer * 8;
+            z-index: 2;
+            display: block;
+            width: $spacer * 16;
+            height: $spacer * 16;
+            border-width: 6px;
+            border-style: solid;
+            border-radius: 50%;
+
+            img {
+
+                display: block;
+                width: 100%;
+                height: 100%;
+                border-radius: 50%;
+            }
+        }
+
+            .r-user__info-travel-mate {
+
+                position: absolute;
+                top: $spacer * 2;
+                left: $spacer * 12;
+            }
+
+        .r-user__info-actions {
+
+            position: relative;
+            z-index: 4;
+            top: -$spacer * 10;
+            padding-left: $spacer * 2.5;
+        }
+
+        .r-user__info-title {
+
+            margin: -$spacer * 7 0 $spacer * 2;
+        }
+
+        .r-user__info-description {
+
+            @include respond-to(tablet) {
+
+                @include span(10 of 24);
+            }
+        }
+
+        .r-user__info-extra {
+
+            position: absolute;
+            z-index: 10;
+            right: 0;
+            bottom: 0;
+
+            @include respond-to(tablet) {
+
+                right: span(2);
+            }
+
+            @include respond-to(desktop) {
+
+                right: $container-desktop/28*2;
+            }
+
+            @include respond-to(desktop-medium) {
+
+                right: $container-desktop-medium/28*2;
+            }
+
+            @include respond-to(desktop-large) {
+
+                right: $container-desktop-large/28*2;
```

Comment:
`$container-desktop-large/28*2` should be written with a single space on each side of the operator: `$container-desktop-large/28 * 2`<br>`$container-desktop-large/28` should be written with a single space on each side of the operator: `$container-desktop-large / 28`